contracts/sol6/KyberHintHandler.sol

```diff
@@ -280,7 +300,7 @@ contract KyberHintHandler is IKyberHint, Utils4 {
         if (error == HintErrors.SplitsNotEmptyError) revert("splits must be empty");
     }
 
-    function convertReserveIdToAddress(bytes32 reserveId) internal view returns (address);
+    function convertReserveIdToAddress(bytes32 reserveId) internal view virtual returns (address);
```

Comment:
we still need this part?
we convert to address?

Reply:
Hmmmm, since we shift to storage, maybe we can read from there instead?

Reply:
We can discuss this. Cos network side also stores ids to address... if we just get from storage, then we can remove it too.